${prefix} setter not present in networking/network

I've been following the landing zone tutorial, but with the `kpt-v1` branch and `1.0.0-beta.1` releases download.

On the [step to create the network](https://cloud.google.com/anthos-config-management/docs/tutorials/landing-zone#create_the_network) I saw an error related to `${prefix}` once the network package was put in place.

The same error reproduces rendering the source package directly.

When the apply-setter is running at the `network` level it seems it isn't picking up the `${prefix}` value from `network/subnet/setters.yaml`?

Steps to reproduce:
```shell
git clone --branch=kpt-v1 https://github.com/GoogleCloudPlatform/blueprints
kpt fn render blueprints/catalog/networking/network
```

```
Package "network/subnet": 
[RUNNING] "gcr.io/kpt-fn/apply-setters:v0.1"
[PASS] "gcr.io/kpt-fn/apply-setters:v0.1"
  Results:
    [INFO] set field value to "network-name-router-nat" in file "subnet/nat.yaml" in field "metadata.name"
    [INFO] set field value to "networking" in file "subnet/nat.yaml" in field "metadata.namespace"
    [INFO] set field value to "project-id" in file "subnet/nat.yaml" in field "metadata.annotations.cnrm.cloud.google.com/project-id"
    [INFO] set field value to "us-central1" in file "subnet/nat.yaml" in field "spec.region"
    ...(13 line(s) truncated, use '--truncate-output=false' to disable)

Package "network/vpc": 
[RUNNING] "gcr.io/kpt-fn/apply-setters:v0.1"
[PASS] "gcr.io/kpt-fn/apply-setters:v0.1"
  Results:
    [INFO] set field value to "project-id-compute" in file "vpc/services.yaml" in field "metadata.name"
    [INFO] set field value to "project-id" in file "vpc/services.yaml" in field "metadata.annotations.cnrm.cloud.google.com/project-id"
    [INFO] set field value to "network-name" in file "vpc/vpc.yaml" in field "metadata.name"
    [INFO] set field value to "networking" in file "vpc/vpc.yaml" in field "metadata.namespace"
    ...(1 line(s) truncated, use '--truncate-output=false' to disable)

Package "network": 
[RUNNING] "gcr.io/kpt-fn/apply-setters:v0.1"
[FAIL] "gcr.io/kpt-fn/apply-setters:v0.1"
  Results:
    [ERROR] failed to apply setters: values for setters [${prefix}] must be provided
  Stderr:
    "values for setters [${prefix}] must be providedvalues for setters [${prefix}] must be provided"
    ""
  Exit code: 1
```

--- a/catalog/networking/network/setters.yaml
+++ b/catalog/networking/network/setters.yaml
@@ -9,3 +9,4 @@ data:
   region: us-central1
   vpn-secret-key: vpn-shared-secret
   vpn-secret-name: vpn-shared-secret
+  prefix: ""

--- a/catalog/networking/network/subnet/setters.yaml
+++ b/catalog/networking/network/subnet/setters.yaml
@@ -4,9 +4,9 @@ metadata:
   name: setters
 data:
   namespace: networking
-  prefix: ""
   ip-cidr-range: 10.2.0.0/16
   network-name: network-name
   project-id: project-id
   region: us-central1
   source-subnetwork-ip-ranges-to-nat: ALL_SUBNETWORKS_ALL_IP_RANGES
+  prefix: ""

